### PR TITLE
Test Degrees and Strengths

### PR DESCRIPTION
**Pull request by egolol**
#26 Created Tests for Degrees and Strengths updated test_braph2 script

### CODE REVIEW

File: braph/graph/measures/test_DegreeAv.m [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    'DegreeAv is not getting class correctly')
+
+%% Test 5: DegreeAv get Description works
+
+g = GraphWU(A);
+average_degree = DegreeAv(g);
+
+assert(ischar(average_degree.getDescription()), ...
+    'BRAPH:DegreeAv:Bug', ...
+    'DegreeAv is not getting class correctly')
```
Comment (giovannivolpe) [inline braph/graph/measures/test_DegreeAv.m]: remove

File: braph/graph/measures/test_DegreeAv.m [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+A = max(A, A');
+degree_test = sum(A, 2);
+average_degree_test = mean(degree_test);
+
+assert(isequal(average_degree.getValue, average_degree_test), ...
+    'BRAPH:DegreeAv:Bug', ...
+    'DegreeAv is not beeing calculated correctly for GraphBU')
+
+%% Test 2: DegreeAv calculates correctly for GraphWU
+
```
Comment (giovannivolpe) [inline braph/graph/measures/test_DegreeAv.m]: remove space

File: braph/graph/measures/test_DegreeAv.m [hunk 3]
```diff
@@ -0,0 +1,60 @@
+% test DegreeAv
+
```
Comment (giovannivolpe) [inline braph/graph/measures/test_DegreeAv.m]: remove space

File: braph/graph/measures/test_DegreeAv.m [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+A = rand(randi(10));
+
+%% Test 1: DegreeAv calculates correctly for GraphBU
+
+g = GraphBU(A);
+average_degree = DegreeAv(g);
+A(1:length(A)+1:end) = 0;
```
Comment (giovannivolpe) [inline braph/graph/measures/test_DegreeAv.m]: add space between these two lines:
average_degree = DegreeAv(g);

A(1:length(A)+1:end) = 0;

File: braph/graph/measures/test_DegreeAv.m [hunk 5]
```diff
@@ -0,0 +1,60 @@
+% test DegreeAv
+
+A = rand(randi(10));
+
+%% Test 1: DegreeAv calculates correctly for GraphBU
+
```
Comment (giovannivolpe) [inline braph/graph/measures/test_DegreeAv.m]: remove space

File: braph/graph/measures/test_DegreeAv.m [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+assert(isequal(average_degree.getValue, average_degree_test), ...
+    'BRAPH:DegreeAv:Bug', ...
+    'DegreeAv is not beeing calculated correctly for GraphBU')
+
+%% Test 2: DegreeAv calculates correctly for GraphWU
+
+g = GraphWU(A);
+average_degree = DegreeAv(g);
+A(1:length(A)+1:end) = 0;
```
Comment (giovannivolpe) [inline braph/graph/measures/test_DegreeAv.m]: add space

File: braph/graph/measures/test_DegreeAv.m [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+A(A<0) = 0;
+A(A>0) = 1;
+A = max(A, A');
+degree_test = sum(A, 2);
+average_degree_test = mean(degree_test);
+
+assert(isequal(average_degree.getValue, average_degree_test), ...
```
Comment (giovannivolpe) [inline braph/graph/measures/test_DegreeAv.m]: add ()

File: braph/graph/measures/test_DegreeAv.m [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+g = GraphBU(A);
+average_degree = DegreeAv(g);
+A(1:length(A)+1:end) = 0;
+A(A<0) = 0;
+A(A>0) = 1;
+A = max(A, A');
+degree_test = sum(A, 2);
+average_degree_test = mean(degree_test);
+
+assert(isequal(average_degree.getValue, average_degree_test), ...
```
Comment (giovannivolpe) [inline braph/graph/measures/test_DegreeAv.m]: add ()

**Comment by egolol** [comment]
updated

**Pull request merged**